pkg/ui/package.json

```diff
@@ -55,7 +55,7 @@
     "@types/react-copy-to-clipboard": "^4.2.5",
     "@types/react-dom": "^16.0.5",
     "@types/react-helmet": "^5.0.5",
-    "@types/react-paginate": "^4.3.1",
+    "@types/react-paginate": "^4.3.2",
```

Comment:
Looks like there are no types for 5.x yet. The page still works with the upgrade, and there are no type errors, so I guess the old types are close enough…